### DISCUSSION TITLE
After creating a Cluster or "Person" can it be shared?

### DISCUSSION THREAD

**Discussion by filipe-silva** [nextcloud/recognize#1136] [Q&A]
Hi everyone, I looked around but couldn't find any way to do this, is it possible?
After creating a Cluster or "Person" have a public link that I can share for a someone to view?

Let's say I have grouped all pictures of Sara, how can i let her see them?

Thanks in advance

**Comment by marcelklehr** [comment]
Hi @filipe-silva 
This is not currently possible, but we are open for contributions :)

**Reply by marcelklehr** [reply]
See https://github.com/nextcloud/recognize/issues/948 for the original issue, you can upvote it to make it more likely that Nextcloud GmbH will work on this
